### PR TITLE
[#10233] Override global styles to have less space around elements

### PR DESCRIPTION
**Pull request by t-cheepeng**
Part of #10233

Overall, most pages have some form of table/card. The classes from bootstrap has slightly too much padding/margin. With these changes, the spaces now look similar to that of V6. This applies site-wide and most of the tables still look okay. Will have to go through each page in more detail to further optimize the space usage after this. Below are some samples of how tables and cards look.

#### Student Home Page Before
image
#### Student Home Page After 
image

#### Session Submission Page Before
image
#### Session Submission Page After
image

#### Session Result Page Before
image
#### Session Result Page After
image

**Outline of Solution**
- Change button texts to be smaller
- Change table padding to be smaller
- Remove margin from table 
- Change card headers padding to be smaller.

### CODE REVIEW

File: src/web/styles.scss [hunk 1]
```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+}
+
+.table th,
+.table td {
+  padding: 0.5rem
```
Comment (Derek-Hardy) [inline src/web/styles.scss:58]: I think group selector can be put on the same line.

Comment (t-cheepeng) [reply src/web/styles.scss:58]: Right, but seems like our ts-ling config requires new lines after the comma

File: src/web/styles.scss [hunk 2]
```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+}
+
+.table th,
+.table td {
+  padding: .5rem;
```
Comment (wkurniawan07) [inline src/web/styles.scss:58]: You can combine this with the previous `.table` style definition. Use the `&` selector.

File: src/web/styles.scss [hunk 3]
```diff
@@ -49,6 +49,19 @@ body {
   padding: 30px 20px;
 }
 
+.table {
+  margin-bottom: 0;
+
+  & th,
```
Comment (wkurniawan07) [inline src/web/styles.scss:55]: On second look, actually the `&` is not at all necessary here because without it the meaning is still the same.

**Pull request merged**